app/Resources/views/macros.html.twig

```diff
@@ -11,3 +11,13 @@
         </span>
     {% endif %}
 {% endmacro %}
+
+{% macro repository_star(repository, is_starred) %}
+    <a href="{{ path('repository_star', {'name': repository.name}) }}" id="repository-star">
+        {% if is_starred %}
+            <span class="glyphicon glyphicon-star"></span>
+        {% else %}
+            <span class="glyphicon glyphicon-star-empty"></span>
+        {% endif %}
+    </a>
```

Comment:
Correct! I shall say even more : `<span class="glyphicon glyphicon-star{{ not is_starred ? '-empty' }}"></span>`